### DISCUSSION TITLE
Calibration and CLEAN

### DISCUSSION THREAD

**Discussion by JSKenyon** [ratt-ru/QuartiCal#189] [General]
Quoting @IanHeywood:

> (This) has been a source of much debate for at least as long as I've been doing radio astronomy. I can remember people arguing 20 years ago about whether to cut AIPS CL tables at the first negative component or not when doing selfcal.
> 
> Definitely don't want artefacts, postive or negative, entering the model. But the argument againt "no negatives whatsoever" is that for real sources you need the negative components in there because of how CLEAN works, things like the loop gain in conjunction with the fact that even unresolved sources might not being registered at the centre of a pixel once an image is formed. The thinking is that negative components are required to best characterise the source once CLEAN has done its thing.
> 
> I've always leant towards the "allow negatives" philosophy, especially when there is a good mask in place (which I maintain is always good practice). Would be keen to hear everyone's thoughts on this.

**Comment by JSKenyon** [comment]
@IanHeywood @Kincaidr @bennahugo I have taken the liberty of moving this to a discussion. Figured it was worth preserving any useful insights.

**Comment by JSKenyon** [comment]
I agree with you @IanHeywood, and I should probably have been a little clearer in my original comment. I was more concerned about actual calibration artifacts entering the model (and the freezing in problem). As you say, CLEAN needs those negative components to effectively describe the source. The exact impact of this on calibration is probably difficult to quantify as simply excluding negative pixels from the model will result in a model which isn't actually consistent with either the sky or the deconvolution procedure. @landmanbester may have some ideas on how this could be improved.

**Comment by landmanbester** [comment]
This is a tough one. I believe that, fundamentally, the need for negative flux stems from insufficient resolution (think of eg. point sources not at the centers of pixels or shock fronts which vary significantly over the extent of a pixel). Also, because clean uses such a simplistic representation of the model, it may actually introduce artifacts even in the absence of calibration errors. In fact any algorithm with a miss-specified prior will do the same. This is part of what makes the deconvolution problem so difficult. I think the most practical solution we have at the moment is to use clean with careful masking and just accept some negative flux. Unfortunately this means you have to be really careful about cleaning thresholds etc. The only alternative I can contemplate, besides using a ridiculously high resolution from the get-go, is to impose positivity a priori under a more physical prior and somehow figure out where higher resolution patches are required. Hence my recent escapades into multi-resolution model formats etc. That's just my 2 cents on the matter. Would be happy to find out that someone else has a simpler solution
